manim/_config/main_utils.py

```diff
@@ -164,6 +164,8 @@ def _parse_args_no_subcmd(args):
 
     parser.add_argument(
         "file",
+        nargs="?",
+        default="",
```

Comment:
What does this do?

Reply:
Makes the file option optional. Before that an error was being raised and I had to run my command uncouth like:

`python -m manim --livestream ""`